### PR TITLE
feat: plugin data injector

### PR DESCRIPTION
**Pull request by blancharda**
Adds a package to allow loading custom keycloak plugin/theme jars at deploytime.

Addresses https://github.com/defenseunicorns/uds-bundle-software-factory-nutanix/issues/54, https://github.com/defenseunicorns/uds-bundle-software-factory-nutanix/issues/55, https://github.com/defenseunicorns/uds-bundle-software-factory-nutanix/issues/56.

(will need to be rebased after we merge https://github.com/defenseunicorns/uds-bundle-software-factory-nutanix/pull/77)

NOTE: in its current form, this will load ALL LOCAL JARS into the keycloak plugin.

### CODE REVIEW

**Comment by blancharda** [comment]
Rough steps for testing:
To test with custom core:
* deploy uds-k3d
* delete the SC and redeploy local-path with reclaim policy set to Retain
* zarf init
* build custom core
* deploy custom core
* VALIDATE no custom plugin
* deploy bundle with –packages keycloak-config-wrapper
* VALIDATE CUSTOM PLUGIN
* re-deploy custom core
* VALIDATE CUSTOM PLUGIN STILL EXISTS

**Comment by blancharda** [comment]
Known TODOs:
* Validate actual PVC will exist for kc providers
    * Match up naming (or expose some vars in the config wrapper pkg)
* Understand topology when KC is in HA mode
    * Match node selectors // affinity for data loader pod

File: config/uds-config.yaml [hunk 1]
```diff
@@ -13,12 +13,15 @@ variables:
   metallb:
     # Replace with a valid IP address range
     IP_ADDRESS_POOL: "10.0.0.10-10.0.0.20"
+  keycloak-configuration:
+    CONFIG_PKG_FULLPATH: "OPTIONAL -- replace with path to keycloak config zarf pkg"
```
Comment (anthonywendt) [inline config/uds-config.yaml:17]: I'm not following where this package and variable are used?

File: config/uds-config.yaml [hunk 2]
```diff
@@ -13,12 +13,15 @@ variables:
...
   core:
     # TLS CERT and KEY values must be base64 encoded
     ADMIN_TLS_CERT: replace-me-cert
     ADMIN_TLS_KEY: replace-me-key
     TENANT_TLS_CERT: replace-me-cert
     TENANT_TLS_KEY: replace-me-key
+    KEYCLOAK_CONFIG_IMAGE: "OPTIONAL -- container image for keycloak config"
```
Comment (anthonywendt) [inline config/uds-config.yaml:24]: I'm not following where this variable is defined?

File: config/uds-config.yaml (hunk 1 above)
Comment (blancharda) [reply config/uds-config.yaml:17]: Whoops!
This is like the 3rd refactor of this branch.. some old stuff seems to have snuck through..

File: config/uds-config.yaml (hunk 2 above)
Comment (blancharda) [reply config/uds-config.yaml:24]: Good catch!
Definitely accidentally leftover from a previous iteration.

**Review by anthonywendt — APPROVED** [review]

**Pull request merged**